Gemfile

```diff
@@ -76,3 +76,5 @@ group :test do
   gem 'webdrivers'
   gem 'webmock'
 end
+
+gem "bundler-audit", "~> 0.9.1", :group => :development
```

Comment:
dependency vulnerabilities, this will fail as there's 2 with issues, but I just want to see how CI handles it